docs/cmake-presets.md

```diff
@@ -0,0 +1,272 @@
+
+CMake Tools uses the value of `hostOS` in the Microsoft Visual Studio Settings vendor map to hide Configure Presets that don't apply to your platform. See the entry for `hostOS` in the table under [Visual Studio Settings vendor map](cmake-presets-json-reference.md#visual-studio-settings-vendor-map) for more information.
+
+## CMake: configure
+
+To configure the project, run **CMake: Configure** from the command palette. This is the same as running `cmake --preset <configurePreset>` from the command line, where `<configurePreset>` is the name of the active Configure Preset.
+
+## CMake: select Build Preset
+
+**CMake: Select Build Preset** lists the Default Build Preset and the union of non-hidden Build Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. The Default Build Preset is equivalent to passing `cmake --build` with no additional arguments from the command line. Select a Build Preset to make it the active Build Preset. This is the `buildPreset` used when CMake is invoked to build the project. The active Build Preset is displayed in the status bar.
+
+All Build Presets are required to specify an associated configurePreset. CMake Tools hides Build Presets that don't apply to the active Configure Preset. For more information, see [Build Presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#build-preset).
+
+## CMake: build
+
+Run **CMake: Build** from the command palette to build the entire project. This is the same as running `cmake --build --preset <buildPreset>` from the command line, where `<buildPreset>` should be replaced by the name of the active Build Preset.
+
+Run **CMake: Build Target** from the command palette to build a single target. You can switch the active build target with **CMake: Set Build Target**.
+
+> [!NOTE]
+> CMake Tools doesn't yet support the **buildPresets.targets** option to build a subset of targets specified in `CMakePresets.json`.
+
+## Test
+
+CTest is the CMake test driver program and is integrated with the CMake Tools extension. For more information, see the [CTest documentation](https://cmake.org/cmake/help/latest/manual/ctest.1.html#ctest-1).
+
+## CMake: select Test Preset
+
+**CMake: Select Test Preset** lists the Default Test Preset and the union of non-hidden Test Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. The Default Test Preset is the same as invoking `ctest` with no other arguments from the command line.
+
+Select a Test Preset to make it the active Test Preset. This is the `testPreset` that will be used when CTest is invoked to run tests. The active Test Preset is displayed in the status bar.
+
+All Test Presets are required to specify an associated `configurePreset`. CMake Tools will hide Test Presets that don't apply to the active Configure Preset. For more information, see [Test Presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#test-preset).
+
+## CMake: run tests
```

Comment:
Capitalize Run Tests (these are all commands that are capitalized in the product)

Reply:
done